.github/ISSUE_TEMPLATE/zenodo_release.md

```diff
@@ -0,0 +1,16 @@
+---
+name: "\U0001F1FF Zenodo release"
+description: Request a Zenodo release
+title: 'Zenodo release: '
+labels: [Zenodo release]
+assignees: ''
+
+---
+
+Can you please create a Zenodo release for my upcoming publication:
+
+### On analytic and algebraic topology of locally Euclidean parameterization of infinitely differentiable Riemannian manifold
```

Comment:
```suggestion
### My Paper Title
```

Reply:
Do you get the reference?

Reply:
Yes, but I don't think it's that helpful